Stop "using WTF::UUID" in the global scope by replacing UUID with WTF::UUID

fujii opened the pull request:
#### d8d3aef8dff8be07e6a3b9f3a22caf4099181ea9
<pre>
Stop &quot;using WTF::UUID&quot; in the global scope by replacing UUID with WTF::UUID
<a href="https://bugs.webkit.org/show_bug.cgi?id=234696">https://bugs.webkit.org/show_bug.cgi?id=234696</a>

Reviewed by Michael Catanzaro.

clang-cl reports &quot;error: reference to &apos;UUID&apos; is ambiguous&quot; for Windows
port. Including &lt;windows.h&gt; defines `UUID` in the global scope. This
conflicted with &quot;using WTF::UUID&quot; in &lt;wtf/Forward.h&gt;.

Changed the style checker rule `build/messagesin/wtf` to allow `WTF::`
prefix of `WTF::UUID`.

* Source/WTF/wtf/Forward.h:
* Source/WebCore/Modules/notifications/Notification.cpp:
(WebCore::Notification::create):
(WebCore::Notification::createForServiceWorker):
(WebCore::Notification::Notification):
(WebCore::Notification::ensureOnNotificationThread):
* Source/WebCore/Modules/notifications/Notification.h:
* Source/WebCore/Modules/notifications/NotificationData.h:
* Source/WebCore/Modules/notifications/NotificationDataCocoa.mm:
(WebCore::NotificationData::fromDictionary):
* Source/WebCore/Modules/push-api/PushDatabase.cpp:
(WebCore::uuidToSpan):
(WebCore::uuidFromSpan):
* Source/WebCore/Modules/push-api/PushSubscriptionIdentifier.h:
(WTF::HashTraits&lt;WebCore::PushSubscriptionSetIdentifier&gt;::constructDeletedValue):
* Source/WebCore/Modules/webdatabase/DatabaseTracker.cpp:
(WebCore::generateDatabaseFileName):
* Source/WebCore/animation/KeyframeEffect.cpp:
(WebCore::KeyframeEffect::KeyframeEffect):
* Source/WebCore/dom/Document.cpp:
(WebCore::Document::originIdentifierForPasteboard const):
* Source/WebCore/dom/UserGestureIndicator.cpp:
(WebCore::UserGestureToken::UserGestureToken):
(WebCore::UserGestureIndicator::UserGestureIndicator):
(WebCore::UserGestureIndicator::authorizationToken const):
* Source/WebCore/dom/UserGestureIndicator.h:
(WebCore::UserGestureToken::create):
(WebCore::UserGestureToken::authorizationToken const):
* Source/WebCore/fileapi/BlobURL.cpp:
(WebCore::BlobURL::createBlobURL):
* Source/WebCore/loader/FetchOptions.h:
(WebCore::FetchOptions::FetchOptions):
* Source/WebCore/loader/appcache/ApplicationCacheStorage.cpp:
(WebCore::ApplicationCacheStorage::writeDataToUniqueFileInDirectory):
* Source/WebCore/platform/PlatformEvent.h:
(WebCore::PlatformEvent::authorizationToken const):
* Source/WebCore/platform/ScriptExecutionContextIdentifier.h:
(WebCore::ProcessQualified&lt;WTF::UUID&gt;::generate):
(WebCore::ProcessQualified&lt;WTF::UUID&gt;::ProcessQualified):
(WebCore::ProcessQualified&lt;WTF::UUID&gt;::object const):
(WebCore::add):
(WebCore::ProcessQualified&lt;WTF::UUID&gt;::decode):
(WebCore::operator&lt;&lt;):
(WebCore::ProcessQualified&lt;UUID&gt;::generate): Deleted.
(WebCore::ProcessQualified&lt;UUID&gt;::ProcessQualified): Deleted.
(WebCore::ProcessQualified&lt;UUID&gt;::operator bool const): Deleted.
(WebCore::ProcessQualified&lt;UUID&gt;::object const): Deleted.
(WebCore::ProcessQualified&lt;UUID&gt;::processIdentifier const): Deleted.
(WebCore::ProcessQualified&lt;UUID&gt;::isHashTableDeletedValue const): Deleted.
(WebCore::ProcessQualified&lt;UUID&gt;::operator== const): Deleted.
(WebCore::ProcessQualified&lt;UUID&gt;::toString const): Deleted.
(WebCore::ProcessQualified&lt;UUID&gt;::encode const): Deleted.
(WebCore::ProcessQualified&lt;UUID&gt;::decode): Deleted.
* Source/WebCore/platform/graphics/ca/GraphicsLayerCA.cpp:
(WebCore::GraphicsLayerCA::updateAnimations):
* Source/WebCore/testing/Internals.cpp:
(WebCore::Internals::isDocumentAlive const):
* Source/WebKit/NetworkProcess/NetworkProcess.cpp:
(WebKit::NetworkProcess::addStorageSession):
* Source/WebKit/NetworkProcess/NetworkSessionCreationParameters.cpp:
(WebKit::NetworkSessionCreationParameters::decode):
* Source/WebKit/NetworkProcess/NetworkSessionCreationParameters.h:
* Source/WebKit/NetworkProcess/Notifications/NetworkNotificationManager.cpp:
(WebKit::NetworkNotificationManager::cancelNotification):
(WebKit::NetworkNotificationManager::clearNotifications):
(WebKit::NetworkNotificationManager::didDestroyNotification):
* Source/WebKit/NetworkProcess/Notifications/NetworkNotificationManager.h:
* Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp:
(WebKit::NetworkStorageManager::create):
(WebKit::NetworkStorageManager::NetworkStorageManager):
* Source/WebKit/NetworkProcess/storage/NetworkStorageManager.h:
* Source/WebKit/NetworkProcess/webrtc/NetworkMDNSRegister.cpp:
(WebKit::NetworkMDNSRegister::registerMDNSName):
* Source/WebKit/Platform/IPC/DaemonCoders.h:
(WebKit::Daemon::Coder&lt;WTF::UUID&gt;::decode):
* Source/WebKit/Scripts/webkit/messages.py:
(headers_for_type):
* Source/WebKit/Shared/NavigationActionData.h:
* Source/WebKit/Shared/NavigationActionData.serialization.in:
* Source/WebKit/Shared/Notifications/NotificationManagerMessageHandler.h:
* Source/WebKit/Shared/Notifications/NotificationManagerMessageHandler.messages.in:
* Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in:
* Source/WebKit/Shared/WebEvent.cpp:
(WebKit::WebEvent::WebEvent):
* Source/WebKit/Shared/WebEvent.h:
(WebKit::WebEvent::authorizationToken const):
* Source/WebKit/Shared/WebEvent.serialization.in:
* Source/WebKit/Shared/WebPushDaemonConnectionConfiguration.h:
(WebKit::WebPushD::WebPushDaemonConnectionConfiguration::decode):
* Source/WebKit/Shared/WebPushDaemonConnectionConfiguration.serialization.in:
* Source/WebKit/Shared/mac/WebEventFactory.mm:
(WebKit::WebEventFactory::createWebMouseEvent):
(WebKit::WebEventFactory::createWebWheelEvent):
(WebKit::WebEventFactory::createWebKeyboardEvent):
* Source/WebKit/UIProcess/API/APIUserInitiatedAction.h:
* Source/WebKit/UIProcess/API/C/WKNotificationManager.cpp:
(WKNotificationManagerProviderDidClickNotification_b):
* Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm:
(+[WKWebsiteDataStore dataStoreForIdentifier:]):
(-[WKWebsiteDataStore setProxyConfigurations:]):
(+[WKWebsiteDataStore _removeDataStoreWithIdentifier:completionHandler:]):
* Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionControllerConfiguration.mm:
(+[_WKWebExtensionControllerConfiguration configurationWithIdentifier:]):
(-[_WKWebExtensionControllerConfiguration initWithCoder:]):
* Source/WebKit/UIProcess/API/Cocoa/_WKWebsiteDataStoreConfiguration.mm:
(-[_WKWebsiteDataStoreConfiguration initWithIdentifier:]):
* Source/WebKit/UIProcess/Cocoa/GroupActivities/WKGroupSession.swift:
(WKGroupSessionWrapper.uuid):
* Source/WebKit/UIProcess/Cocoa/SystemPreviewControllerCocoa.mm:
* Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm:
(WebKit::WebExtensionContext::setUniqueIdentifier):
* Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionControllerConfiguration.mm:
(WebKit::WebExtensionControllerConfiguration::createStorageDirectoryPath):
* Source/WebKit/UIProcess/Extensions/WebExtensionContext.h:
* Source/WebKit/UIProcess/Extensions/WebExtensionControllerConfiguration.cpp:
(WebKit::WebExtensionControllerConfiguration::WebExtensionControllerConfiguration):
* Source/WebKit/UIProcess/Extensions/WebExtensionControllerConfiguration.h:
(WebKit::WebExtensionControllerConfiguration::create):
(WebKit::WebExtensionControllerConfiguration::identifier const):
* Source/WebKit/UIProcess/Launcher/glib/BubblewrapLauncher.cpp:
(WebKit::effectiveApplicationId):
* Source/WebKit/UIProcess/Notifications/ServiceWorkerNotificationHandler.cpp:
(WebKit::ServiceWorkerNotificationHandler::dataStoreForNotificationID):
(WebKit::ServiceWorkerNotificationHandler::cancelNotification):
(WebKit::ServiceWorkerNotificationHandler::clearNotifications):
(WebKit::ServiceWorkerNotificationHandler::didDestroyNotification):
* Source/WebKit/UIProcess/Notifications/ServiceWorkerNotificationHandler.h:
* Source/WebKit/UIProcess/Notifications/WebNotification.cpp:
(WebKit::WebNotification::WebNotification):
* Source/WebKit/UIProcess/Notifications/WebNotification.h:
(WebKit::WebNotification::createPersistent):
(WebKit::WebNotification::coreNotificationID const):
(WebKit::WebNotification::dataStoreIdentifier const):
* Source/WebKit/UIProcess/Notifications/WebNotificationManagerMessageHandler.cpp:
(WebKit::WebNotificationManagerMessageHandler::cancelNotification):
(WebKit::WebNotificationManagerMessageHandler::clearNotifications):
(WebKit::WebNotificationManagerMessageHandler::didDestroyNotification):
* Source/WebKit/UIProcess/Notifications/WebNotificationManagerMessageHandler.h:
* Source/WebKit/UIProcess/Notifications/WebNotificationManagerProxy.cpp:
(WebKit::WebNotificationManagerProxy::cancel):
(WebKit::WebNotificationManagerProxy::didDestroyNotification):
(WebKit::WebNotificationManagerProxy::clearNotifications):
(WebKit::WebNotificationManagerProxy::providerDidClickNotification):
(WebKit::WebNotificationManagerProxy::providerDidCloseNotifications):
* Source/WebKit/UIProcess/Notifications/WebNotificationManagerProxy.h:
* Source/WebKit/UIProcess/WebPageProxy.cpp:
(WebKit::WebPageProxy::cancelNotification):
(WebKit::WebPageProxy::clearNotifications):
(WebKit::WebPageProxy::didDestroyNotification):
* Source/WebKit/UIProcess/WebPageProxy.h:
* Source/WebKit/UIProcess/WebProcessProxy.cpp:
(WebKit::WebProcessProxy::recordUserGestureAuthorizationToken):
(WebKit::WebProcessProxy::userInitiatedActivity):
(WebKit::WebProcessProxy::consumeIfNotVerifiablyFromUIProcess):
* Source/WebKit/UIProcess/WebProcessProxy.h:
* Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm:
(WebKit::WebsiteDataStore::fetchAllDataStoreIdentifiers):
(WebKit::WebsiteDataStore::removeDataStoreWithIdentifier):
(WebKit::WebsiteDataStore::defaultWebsiteDataStoreDirectory):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp:
(WebKit::WebsiteDataStore::existingDataStoreForIdentifier):
(WebKit::WebsiteDataStore::dataStoreForIdentifier):
(WebKit::WebsiteDataStore::cancelServiceWorkerNotification):
(WebKit::WebsiteDataStore::clearServiceWorkerNotification):
(WebKit::WebsiteDataStore::didDestroyServiceWorkerNotification):
(WebKit::WebsiteDataStore::setProxyConfigData):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h:
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStoreConfiguration.cpp:
(WebKit::WebsiteDataStoreConfiguration::WebsiteDataStoreConfiguration):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStoreConfiguration.h:
(WebKit::WebsiteDataStoreConfiguration::create):
(WebKit::WebsiteDataStoreConfiguration::identifier const):
* Source/WebKit/UIProcess/ios/WKModelView.mm:
(-[WKModelView createFileForModel:]):
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundle.cpp:
(WebKit::InjectedBundle::webNotificationID):
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundle.h:
* Source/WebKit/WebProcess/Model/mac/ARKitInlinePreviewModelPlayerMac.mm:
(WebKit::ARKitInlinePreviewModelPlayerMac::createFile):
* Source/WebKit/WebProcess/Notifications/WebNotificationManager.cpp:
(WebKit::WebNotificationManager::didShowNotification):
(WebKit::WebNotificationManager::didClickNotification):
(WebKit::WebNotificationManager::didCloseNotifications):
* Source/WebKit/WebProcess/Notifications/WebNotificationManager.h:
* Source/WebKit/WebProcess/Notifications/WebNotificationManager.messages.in:
* Source/WebKit/webpushd/PushClientConnection.h:
(WebPushD::ClientConnection::dataStoreIdentifier const):
* Source/WebKitLegacy/WebCoreSupport/NetworkStorageSessionMap.cpp:
(NetworkStorageSessionMap::ensureSession):
* Source/WebKitLegacy/mac/WebCoreSupport/WebNotificationClient.h:
* Tools/Scripts/webkitpy/style/checkers/messagesin.py:
(MessagesInChecker.check_WTF_prefix):
* Tools/TestWebKitAPI/Tests/WTF/UUID.cpp:
(parseAndStringifyUUID):
(TEST):
* Tools/TestWebKitAPI/Tests/WTF/cocoa/UUIDCocoa.mm:
(TEST):
* Tools/TestWebKitAPI/Tests/WebCore/PushDatabase.cpp:
(TestWebKitAPI::TEST_F):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/WebPushDaemon.mm:
* Tools/WebKitTestRunner/DataFunctions.h:
(WTR::dataToUUID):
(WTR::uuidToData):
* Tools/WebKitTestRunner/WebNotificationProvider.cpp:
(WTR::WebNotificationProvider::removeNotificationManager):
* Tools/WebKitTestRunner/WebNotificationProvider.h:

Canonical link: <a href="https://commits.webkit.org/265828@main">https://commits.webkit.org/265828@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/259dcc73bac44a01a6f111f77940b1e3ed3639c6

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/11922 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/12121 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/12498 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/13567 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/11352 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/11941 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/14510 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/12102 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/14212 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/12086 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/12903 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/10087 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/13995 "Built successfully") | 
| [✅ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/6/builds/12017 "Passed tests") | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/10195 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/10823 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/17954 "Passed tests") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/10104 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/11273 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/10982 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/14153 "Passed tests") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/11278 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/11394 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/9421 "Passed tests") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/12004 "Built successfully") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/10571 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/3218 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/2936 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/14854 "Built successfully") | | [✅ 🛠 jsc-mips](https://ews-build.webkit.org/#/builders/24/builds/12340 "Built successfully") | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/11250 "Built successfully") | | [✅ 🧪 jsc-mips-tests](https://ews-build.webkit.org/#/builders/3/builds/2983 "Passed tests") | 
<!--EWS-Status-Bubble-End-->

mcatanzaro reviewed (approved):
I agree this is the only reasonable solution to avoid the name conflict. And the style checker change is fine.

Need to make it actually build successfully, though.

fujii committed:
Stop "using WTF::UUID" in the global scope by replacing UUID with WTF::UUID
https://bugs.webkit.org/show_bug.cgi?id=234696

Reviewed by Michael Catanzaro.

clang-cl reports "error: reference to 'UUID' is ambiguous" for Windows
port. Including <windows.h> defines `UUID` in the global scope. This
conflicted with "using WTF::UUID" in <wtf/Forward.h>.

Changed the style checker rule `build/messagesin/wtf` to allow `WTF::`
prefix of `WTF::UUID`.

* Source/WTF/wtf/Forward.h:
* Source/WebCore/Modules/notifications/Notification.cpp:
(WebCore::Notification::create):
(WebCore::Notification::createForServiceWorker):
(WebCore::Notification::Notification):
(WebCore::Notification::ensureOnNotificationThread):
* Source/WebCore/Modules/notifications/Notification.h:
* Source/WebCore/Modules/notifications/NotificationData.h:
* Source/WebCore/Modules/notifications/NotificationDataCocoa.mm:
(WebCore::NotificationData::fromDictionary):
* Source/WebCore/Modules/push-api/PushDatabase.cpp:
(WebCore::uuidToSpan):
(WebCore::uuidFromSpan):
* Source/WebCore/Modules/push-api/PushSubscriptionIdentifier.h:
(WTF::HashTraits<WebCore::PushSubscriptionSetIdentifier>::constructDeletedValue):
* Source/WebCore/Modules/webdatabase/DatabaseTracker.cpp:
(WebCore::generateDatabaseFileName):
* Source/WebCore/animation/KeyframeEffect.cpp:
(WebCore::KeyframeEffect::KeyframeEffect):
* Source/WebCore/dom/Document.cpp:
(WebCore::Document::originIdentifierForPasteboard const):
* Source/WebCore/dom/UserGestureIndicator.cpp:
(WebCore::UserGestureToken::UserGestureToken):
(WebCore::UserGestureIndicator::UserGestureIndicator):
(WebCore::UserGestureIndicator::authorizationToken const):
* Source/WebCore/dom/UserGestureIndicator.h:
(WebCore::UserGestureToken::create):
(WebCore::UserGestureToken::authorizationToken const):
* Source/WebCore/fileapi/BlobURL.cpp:
(WebCore::BlobURL::createBlobURL):
* Source/WebCore/loader/FetchOptions.h:
(WebCore::FetchOptions::FetchOptions):
* Source/WebCore/loader/appcache/ApplicationCacheStorage.cpp:
(WebCore::ApplicationCacheStorage::writeDataToUniqueFileInDirectory):
* Source/WebCore/platform/PlatformEvent.h:
(WebCore::PlatformEvent::authorizationToken const):
* Source/WebCore/platform/ScriptExecutionContextIdentifier.h:
(WebCore::ProcessQualified<WTF::UUID>::generate):
(WebCore::ProcessQualified<WTF::UUID>::ProcessQualified):
(WebCore::ProcessQualified<WTF::UUID>::object const):
(WebCore::add):
(WebCore::ProcessQualified<WTF::UUID>::decode):
(WebCore::operator<<):
(WebCore::ProcessQualified<UUID>::generate): Deleted.
(WebCore::ProcessQualified<UUID>::ProcessQualified): Deleted.
(WebCore::ProcessQualified<UUID>::operator bool const): Deleted.
(WebCore::ProcessQualified<UUID>::object const): Deleted.
(WebCore::ProcessQualified<UUID>::processIdentifier const): Deleted.
(WebCore::ProcessQualified<UUID>::isHashTableDeletedValue const): Deleted.
(WebCore::ProcessQualified<UUID>::operator== const): Deleted.
(WebCore::ProcessQualified<UUID>::toString const): Deleted.
(WebCore::ProcessQualified<UUID>::encode const): Deleted.
(WebCore::ProcessQualified<UUID>::decode): Deleted.
* Source/WebCore/platform/graphics/ca/GraphicsLayerCA.cpp:
(WebCore::GraphicsLayerCA::updateAnimations):
* Source/WebCore/testing/Internals.cpp:
(WebCore::Internals::isDocumentAlive const):
* Source/WebKit/NetworkProcess/NetworkProcess.cpp:
(WebKit::NetworkProcess::addStorageSession):
* Source/WebKit/NetworkProcess/NetworkSessionCreationParameters.cpp:
(WebKit::NetworkSessionCreationParameters::decode):
* Source/WebKit/NetworkProcess/NetworkSessionCreationParameters.h:
* Source/WebKit/NetworkProcess/Notifications/NetworkNotificationManager.cpp:
(WebKit::NetworkNotificationManager::cancelNotification):
(WebKit::NetworkNotificationManager::clearNotifications):
(WebKit::NetworkNotificationManager::didDestroyNotification):
* Source/WebKit/NetworkProcess/Notifications/NetworkNotificationManager.h:
* Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp:
(WebKit::NetworkStorageManager::create):
(WebKit::NetworkStorageManager::NetworkStorageManager):
* Source/WebKit/NetworkProcess/storage/NetworkStorageManager.h:
* Source/WebKit/NetworkProcess/webrtc/NetworkMDNSRegister.cpp:
(WebKit::NetworkMDNSRegister::registerMDNSName):
* Source/WebKit/Platform/IPC/DaemonCoders.h:
(WebKit::Daemon::Coder<WTF::UUID>::decode):
* Source/WebKit/Scripts/webkit/messages.py:
(headers_for_type):
* Source/WebKit/Shared/NavigationActionData.h:
* Source/WebKit/Shared/NavigationActionData.serialization.in:
* Source/WebKit/Shared/Notifications/NotificationManagerMessageHandler.h:
* Source/WebKit/Shared/Notifications/NotificationManagerMessageHandler.messages.in:
* Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in:
* Source/WebKit/Shared/WebEvent.cpp:
(WebKit::WebEvent::WebEvent):
* Source/WebKit/Shared/WebEvent.h:
(WebKit::WebEvent::authorizationToken const):
* Source/WebKit/Shared/WebEvent.serialization.in:
* Source/WebKit/Shared/WebPushDaemonConnectionConfiguration.h:
(WebKit::WebPushD::WebPushDaemonConnectionConfiguration::decode):
* Source/WebKit/Shared/WebPushDaemonConnectionConfiguration.serialization.in:
* Source/WebKit/Shared/mac/WebEventFactory.mm:
(WebKit::WebEventFactory::createWebMouseEvent):
(WebKit::WebEventFactory::createWebWheelEvent):
(WebKit::WebEventFactory::createWebKeyboardEvent):
* Source/WebKit/UIProcess/API/APIUserInitiatedAction.h:
* Source/WebKit/UIProcess/API/C/WKNotificationManager.cpp:
(WKNotificationManagerProviderDidClickNotification_b):
* Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm:
(+[WKWebsiteDataStore dataStoreForIdentifier:]):
(-[WKWebsiteDataStore setProxyConfigurations:]):
(+[WKWebsiteDataStore _removeDataStoreWithIdentifier:completionHandler:]):
* Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionControllerConfiguration.mm:
(+[_WKWebExtensionControllerConfiguration configurationWithIdentifier:]):
(-[_WKWebExtensionControllerConfiguration initWithCoder:]):
* Source/WebKit/UIProcess/API/Cocoa/_WKWebsiteDataStoreConfiguration.mm:
(-[_WKWebsiteDataStoreConfiguration initWithIdentifier:]):
* Source/WebKit/UIProcess/Cocoa/GroupActivities/WKGroupSession.swift:
(WKGroupSessionWrapper.uuid):
* Source/WebKit/UIProcess/Cocoa/SystemPreviewControllerCocoa.mm:
* Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm:
(WebKit::WebExtensionContext::setUniqueIdentifier):
* Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionControllerConfiguration.mm:
(WebKit::WebExtensionControllerConfiguration::createStorageDirectoryPath):
* Source/WebKit/UIProcess/Extensions/WebExtensionContext.h:
* Source/WebKit/UIProcess/Extensions/WebExtensionControllerConfiguration.cpp:
(WebKit::WebExtensionControllerConfiguration::WebExtensionControllerConfiguration):
* Source/WebKit/UIProcess/Extensions/WebExtensionControllerConfiguration.h:
(WebKit::WebExtensionControllerConfiguration::create):
(WebKit::WebExtensionControllerConfiguration::identifier const):
* Source/WebKit/UIProcess/Launcher/glib/BubblewrapLauncher.cpp:
(WebKit::effectiveApplicationId):
* Source/WebKit/UIProcess/Notifications/ServiceWorkerNotificationHandler.cpp:
(WebKit::ServiceWorkerNotificationHandler::dataStoreForNotificationID):
(WebKit::ServiceWorkerNotificationHandler::cancelNotification):
(WebKit::ServiceWorkerNotificationHandler::clearNotifications):
(WebKit::ServiceWorkerNotificationHandler::didDestroyNotification):
* Source/WebKit/UIProcess/Notifications/ServiceWorkerNotificationHandler.h:
* Source/WebKit/UIProcess/Notifications/WebNotification.cpp:
(WebKit::WebNotification::WebNotification):
* Source/WebKit/UIProcess/Notifications/WebNotification.h:
(WebKit::WebNotification::createPersistent):
(WebKit::WebNotification::coreNotificationID const):
(WebKit::WebNotification::dataStoreIdentifier const):
* Source/WebKit/UIProcess/Notifications/WebNotificationManagerMessageHandler.cpp:
(WebKit::WebNotificationManagerMessageHandler::cancelNotification):
(WebKit::WebNotificationManagerMessageHandler::clearNotifications):
(WebKit::WebNotificationManagerMessageHandler::didDestroyNotification):
* Source/WebKit/UIProcess/Notifications/WebNotificationManagerMessageHandler.h:
* Source/WebKit/UIProcess/Notifications/WebNotificationManagerProxy.cpp:
(WebKit::WebNotificationManagerProxy::cancel):
(WebKit::WebNotificationManagerProxy::didDestroyNotification):
(WebKit::WebNotificationManagerProxy::clearNotifications):
(WebKit::WebNotificationManagerProxy::providerDidClickNotification):
(WebKit::WebNotificationManagerProxy::providerDidCloseNotifications):
* Source/WebKit/UIProcess/Notifications/WebNotificationManagerProxy.h:
* Source/WebKit/UIProcess/WebPageProxy.cpp:
(WebKit::WebPageProxy::cancelNotification):
(WebKit::WebPageProxy::clearNotifications):
(WebKit::WebPageProxy::didDestroyNotification):
* Source/WebKit/UIProcess/WebPageProxy.h:
* Source/WebKit/UIProcess/WebProcessProxy.cpp:
(WebKit::WebProcessProxy::recordUserGestureAuthorizationToken):
(WebKit::WebProcessProxy::userInitiatedActivity):
(WebKit::WebProcessProxy::consumeIfNotVerifiablyFromUIProcess):
* Source/WebKit/UIProcess/WebProcessProxy.h:
* Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm:
(WebKit::WebsiteDataStore::fetchAllDataStoreIdentifiers):
(WebKit::WebsiteDataStore::removeDataStoreWithIdentifier):
(WebKit::WebsiteDataStore::defaultWebsiteDataStoreDirectory):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp:
(WebKit::WebsiteDataStore::existingDataStoreForIdentifier):
(WebKit::WebsiteDataStore::dataStoreForIdentifier):
(WebKit::WebsiteDataStore::cancelServiceWorkerNotification):
(WebKit::WebsiteDataStore::clearServiceWorkerNotification):
(WebKit::WebsiteDataStore::didDestroyServiceWorkerNotification):
(WebKit::WebsiteDataStore::setProxyConfigData):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h:
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStoreConfiguration.cpp:
(WebKit::WebsiteDataStoreConfiguration::WebsiteDataStoreConfiguration):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStoreConfiguration.h:
(WebKit::WebsiteDataStoreConfiguration::create):
(WebKit::WebsiteDataStoreConfiguration::identifier const):
* Source/WebKit/UIProcess/ios/WKModelView.mm:
(-[WKModelView createFileForModel:]):
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundle.cpp:
(WebKit::InjectedBundle::webNotificationID):
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundle.h:
* Source/WebKit/WebProcess/Model/mac/ARKitInlinePreviewModelPlayerMac.mm:
(WebKit::ARKitInlinePreviewModelPlayerMac::createFile):
* Source/WebKit/WebProcess/Notifications/WebNotificationManager.cpp:
(WebKit::WebNotificationManager::didShowNotification):
(WebKit::WebNotificationManager::didClickNotification):
(WebKit::WebNotificationManager::didCloseNotifications):
* Source/WebKit/WebProcess/Notifications/WebNotificationManager.h:
* Source/WebKit/WebProcess/Notifications/WebNotificationManager.messages.in:
* Source/WebKit/webpushd/PushClientConnection.h:
(WebPushD::ClientConnection::dataStoreIdentifier const):
* Source/WebKitLegacy/WebCoreSupport/NetworkStorageSessionMap.cpp:
(NetworkStorageSessionMap::ensureSession):
* Source/WebKitLegacy/mac/WebCoreSupport/WebNotificationClient.h:
* Tools/Scripts/webkitpy/style/checkers/messagesin.py:
(MessagesInChecker.check_WTF_prefix):
* Tools/TestWebKitAPI/Tests/WTF/UUID.cpp:
(parseAndStringifyUUID):
(TEST):
* Tools/TestWebKitAPI/Tests/WTF/cocoa/UUIDCocoa.mm:
(TEST):
* Tools/TestWebKitAPI/Tests/WebCore/PushDatabase.cpp:
(TestWebKitAPI::TEST_F):
* Tools/TestWebKitAPI/Tests/WebKitCocoa/WebPushDaemon.mm:
* Tools/WebKitTestRunner/DataFunctions.h:
(WTR::dataToUUID):
(WTR::uuidToData):
* Tools/WebKitTestRunner/WebNotificationProvider.cpp:
(WTR::WebNotificationProvider::removeNotificationManager):
* Tools/WebKitTestRunner/WebNotificationProvider.h:

Canonical link: https://commits.webkit.org/265828@main
